scripts/wheels/windows/configure.bat

```diff
@@ -61,7 +60,7 @@ cmake -G"Visual Studio 16 2019" -H"%KRATOS_SOURCE%" -B"%KRATOS_BUILD%\%KRATOS_BU
 -DCMAKE_C_FLAGS="/MP24 /Gm- /Zm10"                                                          ^
 -DCMAKE_CXX_FLAGS="/MP24 /Gm- /Zm10"                                                        ^
 -DBOOST_ROOT=%BOOST_ROOT%                                                                   ^
--DKRATOS_BUILD_TESTING=OFF                                                                  ^
+-DKRATOS_BUILD_TESTING=ON                                                                   ^
```

Comment:
you sure about this? What would be the benefit for the user? This will only increase size of the binary and compilation times

I dont think this should be part of the wheels. Maybe if at some point the tests are in a separate binary

Reply:
I agree with @philbucher

Reply:
@pooyan-dadvand Ask me for this. Its still something you need?

Reply:
I also think that, no one will run the tests when they install Kratos from wheels.